app/config/config_prod.yml

```diff
@@ -19,7 +19,7 @@ monolog:
             handler:      nested
         nested:
             type:  stream
-            path:  "%kernel.logs_dir%/%kernel.environment%.log"
+            path:  "php://stderr"
```

Comment:
I tend to agree with David here, logging to STDERR by default becomes the de-facto standard and helps a lot with 12-factor style apps and Docker deployments. Of course, this change could also be made on the Symfony Standard Edition and could lead to changes to the documentation if we go down this route.